### PR TITLE
Strings translations policy

### PR DESCRIPTION
**Pull request by orndorffgrant**
## Why is this needed?
This adds explanations around our new support for translations in pro client.

### CODE REVIEW

File: dev-docs/explanations/string_changes_policy.md
Comment (s-makin) [inline dev-docs/explanations/string_changes_policy.md:9]:
```suggestion
for justification on all GitHub PRs that modify the `messages` module of Ubuntu Pro Client.
```

Comment (s-makin) [reply dev-docs/explanations/string_changes_policy.md:9]: I'm just gonna suggest a global search/replace for this :)

File: dev-docs/explanations/string_changes_policy.md
Comment (s-makin) [inline dev-docs/explanations/string_changes_policy.md:16]: When is/was spell-check CI added? Not clear if this is already implemented or due to appear in the future

File: dev-docs/explanations/string_changes_policy.md
Comment (s-makin) [inline dev-docs/explanations/string_changes_policy.md:19]: I'd rephrase this, I'm not sure why (can't put my finger on it exactly) but the wording of this sounds a bit off. I'd suggest something more like:

Change requested by a Product Manager due to the product being renamed or service descriptions being updated.

File: dev-docs/explanations/string_changes_policy.md
Comment (s-makin) [inline dev-docs/explanations/string_changes_policy.md:21]:
```suggestion
When a change is made to a translated string, a best effort will be made to get Canonicalers to fix the translations of the changed string within the PR that introduces the change. However, a lack of complete translation replacements will not block the release of the changes.
```

Comment (s-makin) [reply dev-docs/explanations/string_changes_policy.md:21]: Should we point out that therefore we cannot guarantee that translations are always completely up to date?

Comment (s-makin) [reply dev-docs/explanations/string_changes_policy.md:21]: Maybe something more like:

When a change is made to a translated string, we (Canonical) will endeavour to provide translations for the changed string within the PR that introduces the change. However, these translations are provided on a "best effort" basis and we will not block the release of the changes due to incomplete translations.

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:8]: "We need to maintain tight control" sounds a bit "off". I'd suggest something more like:

That means Canonical must be both careful and specific about the language used in all messages to ensure that we do not provide misleading or incorrect information.

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:19]: "it is not required" - unclear what "it" refers to in this part

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:32]:
```suggestion
Because many of the Pro Client’s messages are commercial in nature, a bad translation could potentially have odd or
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:37]:
```suggestion
- Consider community bugs/issues regarding translation wording/suggestions.
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:39]:
```suggestion
- **Not** accept community PRs directly changing translations.
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:41]: Wording is a bit confusing, I had to read it twice. Suggest something more like:

For a language to be supported, there must be at least one Canonical employee who speaks/reads/understands that language and is willing to review/help maintain the translation.

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:43]:
```suggestion
We will set up a project on https://translations.launchpad.net where only Canonical employees have permission to contribute
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:45]:
```suggestion
from this Launchpad project and integrate them. *Note that this is not done yet.*
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:47]:
```suggestion
# What parts of the Pro Client will get translated?
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:52]:
```suggestion
    - help text (called out because this is handled in a special way in Pro Client)
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:54]:
```suggestion
- Pro-related messages inserted into `apt upgrade`
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:55]:
```suggestion
- Messages inserted into the MOTD
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:58]:
```suggestion
- JSON output of `pro` CLI commands
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:59]:
```suggestion
  - For example: {“status”: “enabled”} – both “status” and “enabled” should remain in English
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:60]:
```suggestion
  - Exception: some values in the JSON are messages intended for humans – these should get translated
```

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:61]:
```suggestion
- CLI flags
```

**Review by s-makin — COMMENTED** [review]
Just a few suggestions/nits

File: dev-docs/explanations/translations.md
Comment (orndorffgrant) [reply dev-docs/explanations/translations.md:45]: Yes - this part isn't done, but I don't want it to block publishing this doc. I'm open to different ways of communicating that though.

File: dev-docs/explanations/string_changes_policy.md
Comment (orndorffgrant) [reply dev-docs/explanations/string_changes_policy.md]: Done :)

File: dev-docs/explanations/string_changes_policy.md
Comment (orndorffgrant) [reply dev-docs/explanations/string_changes_policy.md:21]: That sounds much better, I'll change it tthat

File: dev-docs/explanations/string_changes_policy.md
Comment (orndorffgrant) [reply dev-docs/explanations/string_changes_policy.md:16]: It will be added by the time this goes live :), I'll clarify that in the text

File: dev-docs/explanations/translations.md
Comment (orndorffgrant) [reply dev-docs/explanations/translations.md:19]: Updating it to be clearer

**Comment by orndorffgrant** [comment]
@s-makin Thank you for the review! I've accepted all of your suggested changes and attempted to clarify the confusing parts you called out. Let me know what you think

File: dev-docs/explanations/translations.md
Comment (s-makin) [reply dev-docs/explanations/translations.md]: That's fine, I just wanted to check that this wasn't a placeholder that you'd forgotten :)

File: dev-docs/explanations/translations.md
Comment (s-makin) [inline dev-docs/explanations/translations.md:21]:
```suggestion
files are not required to be installed with Ubuntu Pro Client (and therefore not required in `ubuntu-minimal`).
```

**Review by s-makin — APPROVED** [review]
LGTM :) there's just one little nit in one of the changes, but otherwise I'm happy if you're happy

File: dev-docs/explanations/translations.md
Comment (orndorffgrant) [reply dev-docs/explanations/translations.md:21]: :upside_down_face: I'll fix it

File: dev-docs/howtoguides/spellcheck.md
Comment (s-makin) [inline dev-docs/howtoguides/spellcheck.md:3]: What language(s) does this check?

File: dev-docs/howtoguides/spellcheck.md
Comment (s-makin) [inline dev-docs/howtoguides/spellcheck.md:18]:
```suggestion
`hunspell` has other options including an interactive mode for fixing errors. Check out the [`hunspell` documentation](https://github.com/hunspell/hunspell#documentation) for more information.
```

**Review by s-makin — COMMENTED** [review]
Generally fine with the new instructions - just a couple more nits and then gtg :)

File: dev-docs/howtoguides/spellcheck.md
Comment (orndorffgrant) [reply dev-docs/howtoguides/spellcheck.md:3]: Only english for now.

@s-makin is this better?
```suggestion
We have CI that runs on every PR which looks for spelling errors in our English messages, but it is often useful to run it locally as well.
```

Comment (s-makin) [reply dev-docs/howtoguides/spellcheck.md:3]: Fine by me - just so anyone working on translations doesn't waste their time running it :)

**Comment by orndorffgrant** [comment]
Thank you again @s-makin! Just resolved the last couple nits

**Comment by orndorffgrant** [comment]
@renanrodrigo or @lucasmoura do you want to review before we merge?

**Comment by renanrodrigo** [comment]
I can take a quick look sure

**Review by renanrodrigo — APPROVED** [review]

**Pull request merged**
